client/tests/integration/multisignature_transaction.rs

```diff
@@ -3,14 +3,14 @@ use std::{str::FromStr as _, thread, time::Duration};
 use eyre::Result;
 use iroha_client::{
     client::{self, Client, QueryResult},
-    config::Configuration as ClientConfiguration,
+    config::Config as ClientConfiguration,
```

Comment:
```suggestion
    config::Config as ClientConfig,
```

Reply:
Renamed it _everywhere_ in the workspace, except `KeyGenConfiguration`.